packages/i18n/locales/en.json

```diff
@@ -279,26 +279,34 @@
     },
     "download": {
       "selectCategory": "Categories",
-      "categories": {
-        "prebuilt-installer": "Prebuilt Installer",
-        "prebuilt-binaries": "Prebuilt Binaries",
-        "package-manager": "Package Manager",
-        "source-code": "Source Code"
-      },
       "buttons": {
-        "prebuilt": "Download Node.js {version}",
-        "source": "Download Node.js {version} source"
+        "installer": "{os} Installer (.{extension})",
+        "binary": "Standalone Binary (.{extension})"
       },
       "dropdown": {
         "bitness": "Bitness",
         "os": "Operating System",
         "version": "Version",
-        "platform": "Platform"
+        "platform": "Platform",
+        "packageManager": "Package Manager",
+        "unknown": "Unknown",
+        "platformGroups": {
+          "official": "Recommended (Official)",
+          "unofficial": "Community (Unofficial)"
+        }
       },
       "codeBox": {
-        "unsupportedVersionWarning": "This version is out of maintenance. Find more information here:",
-        "communityWarning": "Package managers and their installation scripts are not maintained by the Node.js project.",
-        "communityWarningReport": "If you encounter issues, please reach out to the package manager's maintainers."
+        "unsupportedVersionWarning": "This version is out of maintenance. Please use a currently supported version.",
```

Comment:
we should have a link on the box.
also remove translated key because now it's till write "This version is out of maintenance. Find more information here:" so it's strange